app/gateway/2.6.x/configure/auth/oidc-azuread.md

```diff
@@ -129,5 +129,5 @@ Similarly, setting `authenticated_groups_claim` will extract that claim's value
 [add-certificate]: /gateway/{{page.kong_version}}/admin-api/#add-certificate
 [add-service]: /gateway/{{page.kong_version}}/admin-api/#service-object
 [oidc-id-token]: http://openid.net/specs/openid-connect-core-1_0.html#IDToken
-[credential-claim]: https://docs.konghq.com/hub/kong-inc/openid-connect/#configcredential_claim
+[credential-claim]: hub/kong-inc/openid-connect/#configcredential_claim
```

Comment:
```suggestion
[credential-claim]: /hub/kong-inc/openid-connect/#configcredential_claim
```

Reply:
totally unrelated to your changes, but It looks like this list is being rendered.
should it be a bullet list of links?